code/game/machinery/camera/camera.dm

```diff
@@ -50,6 +50,9 @@
 
 	var/internal_light = TRUE //Whether it can light up when an AI views it
 
+	/// A copy of the last paper object that was shown to this camera.
+	var/obj/item/paper/last_shown_paper
```

Comment:
This seems like the source for some pretty nasty hard dels. If the paper gets deleted after being shown, this is set to null via code somewhere?

Reply:
Just noticed it stores copies, so it shouldn't happen in theory. Hopefully fire doesn't burn the contents of cameras or something dumb